### PR TITLE
Make Uglifier behave nicely when compressing ES6 JS

### PR DESCRIPTION
**Pull request by alexsoble**
Reverts studentinsights/studentinsights#1027.

Before I merge, need to fix this issue blocking deploy to Heroku:

```
remote:        rake aborted!
remote:        ExecJS::RuntimeError: SyntaxError: Unexpected token: punc ())
remote:        JS_Parse_Error.get ((execjs):3538:621)
remote:        (execjs):4060:47
remote:        (execjs):1:102
remote:        Object.<anonymous> ((execjs):1:120)
remote:        Module._compile (module.js:570:32)
remote:        Object.Module._extensions..js (module.js:579:10)
remote:        Module.load (module.js:487:32)
remote:        tryModuleLoad (module.js:446:12)
remote:        Function.Module._load (module.js:438:3)
remote:        Module.runMain (module.js:604:10)
remote:        /tmp/build_78e3eab438d06055df53be90ba0b637a/vendor/bundle/ruby/2.3.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:39:in `exec'
remote:        /tmp/build_78e3eab438d06055df53be90ba0b637a/vendor/bundle/ruby/2.3.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:21:in `eval'
remote:        /tmp/build_78e3eab438d06055df53be90ba0b637a/vendor/bundle/ruby/2.3.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:46:in `call'
```

### CODE REVIEW

**Comment by alexsoble** [comment]
Looks like this is what's blocking: https://github.com/browserify-rails/browserify-rails/issues/137

File: config/environments/production.rb [hunk 1]
```diff
@@ -19,7 +19,8 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = Uglifier.new(harmony: true) # ES6 support
```
Comment (kevinrobinson) [inline config/environments/production.rb]: @alexsoble @jhilde This looks like the problem - no bueno for IE11.  Can we just revert this PR to bring the site back up?

**Comment by kevinrobinson** [comment]
@alexsoble one other thing I noticed poking around here is that there's JS included in the PDF export - those might need to either stay plain JS or be included in the build process that compiles down arrow functions, etc.

File: config/environments/production.rb (hunk 1 above)
Comment (pobing) [reply config/environments/production.rb]: 👍

**Pull request merged**